test/metabase/collections_rest/api_test.clj

```diff
@@ -3202,3 +3202,24 @@
   (mt/with-temp [:model/Collection {a-id :id} {:archived true}]
     (is (= "You don't have permissions to do that."
            (mt/user-http-request :rasta :delete 403 (str "/collection/" a-id))))))
+
+(deftest ^:parallel children-sort-clause-description-test
```

Comment:
There are other tests prefixed with `children-sort-clause-test-*`, should you put these 2 there instead? So every sort test is co-located.